LayoutTests/http/wpt/pointerlock/pointerlock_without_gesture.html

```diff
@@ -0,0 +1,73 @@
+<!DOCTYPE html>
+<html>
+<body>
+<meta name="timeout" content="long">
+<script src="/resources/testharness.js"></script>
+<script src="/resources/testharnessreport.js"></script>
+<script src="/resources/testdriver.js"></script>
+<script src="/resources/testdriver-actions.js"></script>
+<script src="/resources/testdriver-vendor.js"></script>
+<style type="text/css">
+    button {
+        color: blue;
+    }
+
+    #target-wrap {
+        position: relative;
+        background-color: lightgrey;
+        width: 200px;
+        height: 100px;
+        border: grey 1px solid;
+    }
+
+    #target {
+        position: relative;
+        background-color: lightyellow;
+        width: 100px;
+        height: 30px;
+        border: yellow 1px solid;
+    }
+
+    #status-log {
+        margin: 10px 0;
+        color: green;
+    }
+</style>
+</head>
+<body>
+    <h2>Description</h2>
+    <p>This test validates that pointer lock does not work without user activation.</p>
+    <hr/>
+
+    <div id="target-wrap">
+        <div id="status-log">Click the "lockTarget" button.</div>
+        <div id="target">Target</div>
+    </div>
+    <hr/>
+
+    <div id="log"></div>
+
+    <script type="text/javascript" >
+        async_test(t => {
+            let target = document.getElementById('target');
+            document.addEventListener('pointerchange', t.unreached_func("Must not acquire pointer lock."));
+
+            // Request pointer lock twice to ensure two failing promises are returned and both are rejected.
+            let p1 = target.requestPointerLock();
+            let p2 = target.requestPointerLock();
+
+            assert_true(p1 && typeof p1.then === 'function', "requestPointerLock first call returns promise.");
```

Comment:
Maybe `p1 instanceof Promise` instead? Checking if it's thenable doesn't quite seem right.

Reply:
I can just eliminate both of these assertions after I take your next suggestion to use `await promise_rejects_dom(...)`.